sycl-jit/jit-compiler/utils/generate.py

```diff
@@ -33,6 +34,9 @@ def main():
         )
 
         def process_file(file_path):
+            # We only need .bc files from libdevice:
+            if re.search(r"[/\\]libsycl-.*\.(o|obj|spv)$", file_path):
```

Comment:
would it be simpler to just check if it ends in `.bc` than to check if it is not `bc`?